backend/db/database_test.go

```diff
@@ -61,14 +80,14 @@ func TestDatabase_GameID(t *testing.T) {
 
 func TestDatabase_Tags(t *testing.T) {
 	_db, _ := NewDatabaseFromEnv()
-	id0, _ := _db.AddGame(game0)
-	id1, _ := _db.AddGame(game1)
+	id0, _ := _db.AddGame(game2)
+	id1, _ := _db.AddGame(game3)
 
-	game0.Id = id0
-	game1.Id = id1
+	game2.Id = id0
+	game3.Id = id1
 
-	res0, _ := _db.GetGamesByTags([]string{"tag0"}, false)
-	res1, _ := _db.GetGamesByTags([]string{"tag1"}, false)
+	res0, _ := _db.GetGamesByTags([]string{"tag100"}, false)
```

Comment:
Why did we change which games are tested?

Reply:
ohh i am sorry the delete function was not working so i made 4 total game object to test and now i realised that only two is enough so don't need to change the previous game object. Delete was not working which i realised late, i will change it and push it